src/localization/localization.ts

```diff
@@ -0,0 +1,20 @@
+import i18n, {Callback, InitOptions} from 'i18next';
+import {initReactI18next} from 'react-i18next';
+
+import {Languages} from './languages';
+
+class Localization {
+  public changeLanguage = async (key: Languages) =>
```

Comment:
you should init this class by i18n initialisation from another class move here impl from initialisation class ...